v4/export/sql_type.go

```diff
@@ -18,37 +18,42 @@ var (
 )
 
 func initColTypeRowReceiverMap() {
-	for _, s := range dataTypeString {
+	dataTypeStringArr := []string{
```

Comment:
please add a comment where we collect those types

Reply:
https://github.com/pingcap/dumpling/pull/273/commits/bfaf7dc29249eeb6b12a071c235f9934387f6a58